number_box: 300
distance_hand: 0.02
discrete_side: 10